fixtures/beamz/mhl-74.json

```diff
@@ -0,0 +1,134 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "MHL-74",
+  "categories": ["Moving Head"],
+  "meta": {
+    "authors": ["Helfire"],
+    "createDate": "2023-10-24",
+    "lastModifyDate": "2023-10-24"
+  },
+  "links": {
```

Comment:
This is the web page, right?  https://www.beamzlighting.com/product/mhl74-mini-moving-head-7x-10w-4-in-1-led/